### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by elizy88**
## Description
Modified the Readme to have a descriptive installation process for Windows.

What does the PR do?
The PR  explains the process of setting the environment for windows OS users or for people who use MAC but want to use Android Studio

### CODE REVIEW

File: app/views/MainView/MainView.js [hunk 1]
```diff
@@ -510,7 +517,7 @@ export default class MainView extends Component {
           scrollEventThrottle={16}
           snapToAlignment="center"
           style={{ overflow: 'visible' }}
-          data={approvedRides}
+          data={outsideAvailRides}
```
Comment (kidatsy) [inline app/views/MainView/MainView.js:520]: @elizy88 it looks like your previous work on the rides outside of the user's availabilty got included in this PR. Can you make sure it's just the README stuff in this one?

**Review by kidatsy — CHANGES_REQUESTED** [review]
README stuff looks fine, but there are a couple issues here:
1. What's going on with all those empty icon font files?
2. It's also including your work on the ticket Nick has now, with the rides outside of the user's availability.

File: app/views/MainView/MainView.js (hunk 1 above)
Comment (elizy88) [reply app/views/MainView/MainView.js]: I have modified the Readme file by adding files for x code installation in Mac , and also I removed the changes I had for the outside my availablity codes.
but I am not clear  about this 
"What's going on with all those empty icon font files?"

Comment (kidatsy) [reply app/views/MainView/MainView.js]: It looks like these files got emptied out somehow 
Screen Shot 2021-03-05 at 9 51 47 AM

Comment (elizy88) [reply app/views/MainView/MainView.js]: @kidatsy  It seems that the assets folder was missing so I have added the assets folder and the subfolder fonts.

**Review by kidatsy — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -9,12 +9,42 @@ Cross-platform mobile app for Android and iOS written with React Native.
...
 ## First time setup
 
 ### Mac
 
-`yarn setup:mac`
+- Install Xcode version 11.3.1 or newer.
+  IF YOU NEED AN OLDER VERSION THEN CLICK HERE: https://developer.apple.com/develop/
+- Use this to install homebrew
+  `/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)`
+- brew install node
```
Comment (kidatsy) [inline README.md:23]: I would consider installing something like `nodenv`, which will allow you to switch between different versions of node: https://github.com/nodenv/nodenv

**Pull request merged**